File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;
    do
    {
        h = get_int("Height: "); //prompt user for number
    }
    while (h < 1 || h > 8); //repeat until number fits parameters
    for (int l = 0; l < h; l++)
    {
        for (int d = l; d < h - 1; d++) //spaces on left 
        {
            printf(" ");
        }
        {
            for (int f = 0; f <= l; f++) //left side of pyramid
            {
                printf("#");
            }
        }
        for (int g = 0; g < 1; g++) //spaces inbetween pyramids
        {
            printf("  ");
        }
        for (int n = 0; n <= l; n++) //right pyramid
        {
            printf("#");
        }
        printf("\n");
    }

} //end

File: mario.py
from cs50 import get_int

while True:
    n = get_int("Height: ")  # prompt user for number
    if n > 0 and n < 9:  # repeats until number fits parameters
        break
for i in range(n)  # rows
    print(" " * ((n - i)-1), end="")  # prints spaces
    print("#" * (i + 1))  # prints #
